### PR TITLE
Preloading Membrane textures

### LINKED ISSUE #1009: Make MembraneType load the textures
Instead of membrane.
https://github.com/Revolutionary-Games/Thrive/blob/bf37ff573abcdafe05017db82999c9669449d83c/src/microbe_stage/Membrane.cs#L370

### PR DESCRIPTION
**Pull request by CoreyHendrey**
Attempting to implement issue #1009 .

I copied the way the preloading was done for Ogranelles - by loading them in the SimulationParameters Resolve section.

I've never contributed to open source before, so I apologize if I've gone about it incorrectly. (I did read all the documentation!)

closes #1009

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
ok to test

File: src/microbe_stage/Membrane.cs [hunk 1]
```diff
@@ -369,8 +369,8 @@ private void ApplyTextures()
             return;
 
         // TODO: add the loaded Texture objects to be in the membrane type
```
Comment (hhyyrylainen) [inline src/microbe_stage/Membrane.cs:371]: Remove this todo comment

Comment (CoreyHendrey) [reply src/microbe_stage/Membrane.cs]: Done

**Comment by CoreyHendrey** [comment]
Woops, I didn't mean to Update Branch.

**Comment by hhyyrylainen** [comment]
I was going to do the update branch anyway as Github wants that to be done before saying it's fine to go ahead with the merge.
Thanks for the contribution.

**Pull request merged**